client/src/components/CreatePost/Form/AskHelp.js

```diff
@@ -32,7 +32,7 @@ const AskHelp = ({
         <Submit
           primary="true"
           onClick={handleSubmit}
-          disabled={!formData.title || !formData.description}
+          disabled={!formData.title || !formData.description || selectedTags < 1}
```

Comment:
@idjevm  just checking, is `selectedTags` a number?  or do you need to use `.length` on an array or something?

Reply:
It is an empty array if there are no tags. There are a few ways I can check for this though. I am not sure which one is better over the other ones...
I could do any of the following: 
```
selectedTags == 0 // true if empty
selectedTags < 1 // true if empty
seledctedTags.length === 0 // true if empty
selectedTags.length < 1 // true if empty

```

Maybe I should change it to `selectedTags.length === 0`?

Reply:
yes it's not going to work if it's an array and you don't use `.lengh` property
`[] < 1` makes no sense

Reply:
you can even use `if ( !selectedTags.length )` which will be evaluated as `false` if 0

Reply:
I mean `[] < 1` worked but weirdly, I guess it is because, behind the scenes, that empty array becomes a string ? or a string 0? But if you evaluate and empty array with < 1 would return true. 

I think you meant `!selectedTags.length` will be evaluated as `true` if 0... 

I updated it to `selectedTags.length === 0` but I could change it to `!selectedTags.length` as well. Both ways should work well though.

Reply:
yes lol i mean `true` :) but i mean `[].length` if 0 is "falsy" in JS terms... but equals 0 is fine too